About/Performance/index.html

```diff
@@ -45,18 +45,18 @@ <h2 id="fastest">The world's fastest kernel</h2>
 <h3>Why IPC Benchmarks?</h3>
 
 <p>
-  In microkernel-based systems, the division of software into separate
-  components is greater than on monolithic operating systems. Usually these
-  components run in separate address spaces. This is excellent for improving
-  security, but can incur overhead if done incorrectly.
+  Microkernel-based systems tend to divide up software into a greater number of
+  components than monolithic systems. Usually these components run in separate
+  address spaces. This is excellent for improving security, but can incur
+  overhead if done incorrectly.
```

Comment:
This bit still needs to change (the `incorrectly` is still wrong), but we can merge this PR and think about that some more later.